### ISSUE TITLE
I get notifications of user activity when there seems to be none

### ISSUE THREAD

**Issue by joshgoebel** [exercism/exercism#5633]
<img width="987" alt="Screen Shot 2021-09-01 at 3 39 43 PM" src="https://user-images.githubusercontent.com/6473/131733639-a8b64f7a-fc71-4c10-a6c8-04f8a68d34e4.png">

Yet I can find no new messages or iterations since my last comment...

<img width="611" alt="Screen Shot 2021-09-01 at 3 40 55 PM" src="https://user-images.githubusercontent.com/6473/131733763-0e7da23b-ed57-4b51-94db-8b9b1b47343b.png">

https://exercism.org/mentoring/discussions/76e7762c-08c7-11ec-81c5-853579c25f94

**Comment by joshgoebel** [comment]
Also the timestamps in notifications dropdown don't correspond to those in Inbox.

**Comment by joshgoebel** [comment]
And now those same conversations are wrong in the opposite way.

The inbox says "20 hours ago" but I click into the thread and there is a new message "4 hours ago".

**Comment by iHiD** [comment]
@joshgoebel I'm not sure I quite understand what you're reporting here. Is it that "3 minutes ago" in the top line is incorrect?

**Comment by joshgoebel** [comment]
Yes, and the 5 min... clicking into those showed no recent activity they were both 20 or 30m old...

**Comment by iHiD** [comment]
Is this still an issue?

**Comment by kytrinyx** [comment]
Is this the same issue as https://github.com/exercism/exercism/issues/6198 by any chance?

**Comment by kytrinyx** [comment]
@joshgoebel do you know if this is still an issue?
